engine/gfx/sprite_palettes.asm

```diff
@@ -1,3 +1,55 @@
+LoadWeatherPalNoApply:
+	ld hl, wPalFlags
+	set NO_DYN_PAL_APPLY_ONCE_F, [hl]
+LoadWeatherPal::
+	ld a, [wCurrentWeather]
+	assert OW_WEATHER_NONE == 0
+	and a
+	ret z
+	assert OW_WEATHER_RAIN == 1
+	dec a
+	jr z, .rain
+	assert OW_WEATHER_SNOW == 2
+	dec a
+	jr z, .snow
+	assert OW_WEATHER_THUNDERSTORM == 3
+	dec a
+	jr z, .rain ; thunderstorm
+	assert OW_WEATHER_SANDSTORM == 4
+	jr .sandstorm
+.snow
+	ldh a, [rSVBK]
+	push af
+	ld a, BANK(wOBPals1)
+	ldh [rSVBK], a
+	ld a, -1
```

Comment:
This is that special "keep existing palette" value, right? Can it be a constant if it isn't already? And be used in data/sprite_anims/oam.asm?

Reply:
No it's being used to tell dynamic pal system that the pal slot doesn't have a palette assigned to it.. we do that cause we are loading an "unofficial" palette (all white for snow). I made a constant.